src/pages/SubmitWorkflow.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { headerBar } from 'src/components/common'
+import { WorkflowInputs } from 'src/pages/WorkflowInputs'
+import { WorkflowSource } from 'src/pages/WorkflowSource'
+
+
+export const SubmitWorkflow = () => {
+  // State
+  const [workflowUrl, setWorkflowUrl] = useState()
+  const [workflowInputs, setWorkflowInputs] = useState()
+  const [showInputsPage, setShowInputsPage] = useState(false)
+
+  return div([
+    headerBar(),
+    div({ style: { margin: '4rem' } }, [
+      h2(['Submit a workflow']),
+      div(['Submit your Terra workflows with the Cromwell engine. Full featured workflow submissions coming soon!']),
+      div({ style: { marginTop: '2rem' } }, [
+        !showInputsPage && h(WorkflowSource, { workflowUrl, setWorkflowUrl, onClick: () => setShowInputsPage(true) }),
+        showInputsPage && h(WorkflowInputs, { workflowUrl, workflowInputs, setWorkflowInputs, displayWorkflowUrlPage: () => setShowInputsPage(false) })
```

Comment:
TOL: It feels to me like the `showInputsPage` decision (and therefore any buttons to change its value?) really belong at the "SubmitWorkflow" component level rather than being passed down as pre-staged functions to sub-components. (eg maybe we could pass the functions to fetch and update `workflowUrl` to the `WorkflowSource` page, but the button which progresses *this* page from its "enter URL" to its "enter inputs" state could be in this component?) What do you think?

Reply:
To confirm my understanding, instead of having the child components directly update `workflowUrl` using `setWorkflowUrl`, we should pass functions to child component that are defined in parent component, and they are called by the child component? Here I was following the pattern similar to Terra UI's [Upload.js component](https://github.com/DataBiosphere/terra-ui/blob/69c85b640ec1a4a41f5388e6c7155ae35e516464/src/pages/Upload.js#L729-L918), where the child components are passed [state setters](https://github.com/DataBiosphere/terra-ui/blob/69c85b640ec1a4a41f5388e6c7155ae35e516464/src/pages/Upload.js#L873-L880) as well as [pre-staged functions](https://github.com/DataBiosphere/terra-ui/blob/69c85b640ec1a4a41f5388e6c7155ae35e516464/src/pages/Upload.js#L803)

Reply:
As discussed, I have moved the buttons to the parent component `SubmitWorkflow`. Now `SubmitWorkflow` will have all the orchestration related code.